testing/test_capture.py

```diff
@@ -851,7 +853,7 @@ def test_write_bytes_to_buffer(self):
 
 
 def test_bytes_io():
-    f = py.io.BytesIO()
+    f = io.BytesIO()
```

Comment:
This test only tests code from the stdlib and can be removed IMO.

Reply:
lol true, want to make a patch for it?

Reply:
Sure. I'll create a PR